packages/hdwallet-ledger/src/ledger.ts

```diff
@@ -327,7 +327,20 @@ export class LedgerHDWallet implements core.HDWallet, core.BTCWallet, core.ETHWa
   }
 
   public async getFirmwareVersion (): Promise<string> {
-    const { version } = await this.getFeatures()
+    await this.transport.open()
+
+    var version
```

Comment:
slightly better (subjective) but it also eliviates the use of _var_ which is not great in JS because of all the scoping rules.  As much as possible we should use _let_ and _const_.
```js
    let version
    try {
        version = (await this.getFeatures()).version
    } catch(err) {
        if (err.message.includes('0x6d00'))
            throw new core.WrongApp('Ledger', 'Dashboard')

        console.error(err)
    } finally {
      await this.transport.close()
    }
```

Reply:
this whole try/catch should go inside `getFeatures()`, no?

Reply:
@keepkeyjon yeah that would make more sense, also cleans up the `.version` piece. Fixed.